src/main/kotlin/no/nav/familie/ef/sak/vilkår/dto/VilkårDto.kt

```diff
@@ -23,6 +23,7 @@ data class VilkårGrunnlagDto(
     val registeropplysningerOpprettetTid: LocalDateTime,
     val adresseopplysninger: AdresseopplysningerDto?,
     val dokumentasjon: DokumentasjonFraSøknadDto?,
+    val harAvsluttetArbeidsforhold: Boolean?,
```

Comment:
Kan den være nullable ?

Reply:
Ja, men det for bakoverkompabilitet. Feltet kan være null i de datagrunnlagene som er lagret allerede og hvor feltet ikke eksisterer.